### PR TITLE
chore(all): migrate apps and packages to expo 52

### LINKED ISSUE #683: Migrate apps and packages to Expo 52
Spoiler from progress: world of pain

### PR DESCRIPTION
**Pull request by HaakonSvane**
- Migrate Chronicles to Expo 52
- Require Expo 52 compatible packages in all repo modules
- Install and configure syncpack for better handling dependencies in repo
- Move some commonly installed packages (in root) to respective sub-packages instead of
- Upgrade Yarn, turborepo and lots of common toolings
- Rewrite drawing engine of drawing package to use react native gesture handler instead (old method was removed by skia 😮)
- Move calculated proxy-token to eds context. Should improve performance quite a lot

closes #683

### CODE REVIEW

**Review by tormodAase — APPROVED** [review]
Looks good! Tested the branch on iPad simulator, and everything seems to run well!

Left a few comments

File: apps/chronicles/package.json [hunk 1]
```diff
@@ -1,72 +1,74 @@
...
+        "expo-status-bar": "~2.0.0",
+        "expo-system-ui": "~4.0.3",
+        "expo-web-browser": "~14.0.1",
+        "react": "18.3.1",
+        "react-dom": "18.3.1",
+        "react-native": "0.76.2",
         "react-native-device-info": "^10.9.0",
-        "react-native-gesture-handler": "~2.16.1",
+        "react-native-gesture-handler": "~2.20.2",
         "react-native-msal": "git+https://github.com/equinor/react-native-msal.git#db8a2031d6464208c0f9afb5b40e218f9e607de0",
```
[the remark this answers is not in the document]
Comment (tormodAase) [reply apps/chronicles/package.json:38]: Tested removing `react-native-msal`. Removing it makes `mad-auth` unable to build. So maybe keeping it is not a bad idea afterall

File: apps/chronicles/app.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
         "name": "chronicles",
         "slug": "chronicles",
         "version": "1.0.0",
+        "newArchEnabled": false,
```
[the remark this answers is not in the document]
Comment (tormodAase) [reply apps/chronicles/app.json:6]: I tested with new architecture enabled, and everything seemed to run fine! However, I don't know what the best approach is. Enabling new architecture to test it out and make sure our packages are compatible with it, or sticking to the old architecture to make sure our packages are compatible with our current, old architecture apps 🤔

[the remark this answers is not in the document]
Comment (HaakonSvane) [reply apps/chronicles/app.json:6]: Good catch. Forgot I set this after debugging some crashes I thought were related to it. Enabled it and seems to be working for me as well.
I think we should stick to having the new arch enabled in this repo. This way we can at least guarantee that everything from the mobile platform is compatable with it. Hopefully shouldn't take long before we get the other apps ported over as well (knock on wood zebra :) )

File: apps/chronicles/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (HaakonSvane) [reply apps/chronicles/package.json:38]: Yeah I think we need to keep it installed somewhere in the repo as long as it is a peer dep. in a package :/

**Pull request merged**